### ISSUE TITLE
deactivate-r-base.sh not installed with `r-base`

### ISSUE THREAD

**Issue by PertuyF** [conda-forge/r-base-feedstock#70]
Issue:

Conda environment with `r-base` sets environment variable `RSTUDIO_WHICH_R` from [`$CONDA_PREFIX/etc/conda/activate.d/activate-r-base.sh`](https://github.com/conda-forge/r-base-feedstock/blob/master/recipe/activate-r-base.sh), but does not reset to initial state upon deactivation.
This state will keep `RSTUDIO_WHICH_R` set to a specific environment install or R, which can crash `rstudio` in some cases.

Although the [deactivate script](https://github.com/conda-forge/r-base-feedstock/blob/master/recipe/deactivate-r-base.sh) exists in the repo, its installation was [removed from `build.sh`](https://github.com/conda-forge/r-base-feedstock/compare/6b81c0f..HEAD#diff-44a73bcc045c193c3bd45da87994b03bL359) a few months ago.

PR #69 is intended to fix this.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /local/conda/envs/R-test:
#
# Name                    Version                   Build  Channel
_r-mutex                  1.0.0               anacondar_1
binutils_impl_linux-64    2.31.1               h6176602_1    conda-forge
binutils_linux-64         2.31.1               h6176602_3    conda-forge
bwidget                   1.9.11                        1
bzip2                     1.0.6             h14c3975_1002    conda-forge
ca-certificates           2018.11.29           ha4d7672_0    conda-forge
cairo                     1.14.12           h80bd089_1005    conda-forge
curl                      7.64.0               h646f8bb_0    conda-forge
fontconfig                2.13.1            h2176d3f_1000    conda-forge
freetype                  2.9.1             h94bbf69_1005    conda-forge
gcc_impl_linux-64         7.3.0                habb00fd_1    conda-forge
gcc_linux-64              7.3.0                h553295d_3    conda-forge
gettext                   0.19.8.1          h9745a5d_1001    conda-forge
gfortran_impl_linux-64    7.3.0                hdf63c60_1    conda-forge
gfortran_linux-64         7.3.0                h553295d_3    conda-forge
glib                      2.56.2            had28632_1001    conda-forge
graphite2                 1.3.13            hf484d3e_1000    conda-forge
gxx_impl_linux-64         7.3.0                hdf63c60_1    conda-forge
gxx_linux-64              7.3.0                h553295d_3    conda-forge
harfbuzz                  1.9.0             he243708_1001    conda-forge
icu                       58.2              hf484d3e_1000    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
krb5                      1.16.3            hc83ff2d_1000    conda-forge
libcurl                   7.64.0               h01ee5af_0    conda-forge
libedit                   3.1.20170329      hf8c457e_1001    conda-forge
libffi                    3.2.1             hf484d3e_1005    conda-forge
libgcc-ng                 7.3.0                hdf63c60_0    conda-forge
libgfortran-ng            7.3.0                hdf63c60_0
libiconv                  1.15              h14c3975_1004    conda-forge
libpng                    1.6.36            h84994c4_1000    conda-forge
libssh2                   1.8.0             h1ad7b7a_1003    conda-forge
libstdcxx-ng              7.3.0                hdf63c60_0    conda-forge
libtiff                   4.0.10            h648cc4a_1001    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.8             h143f9aa_1005    conda-forge
make                      4.2.1             h14c3975_2004    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
openssl                   1.0.2p            h14c3975_1002    conda-forge
pango                     1.40.14           hf0c64fd_1003    conda-forge
pcre                      8.41              hf484d3e_1003    conda-forge
pixman                    0.34.0            h14c3975_1003    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
r-base                    3.5.1             he45234b_1005    conda-forge
readline                  7.0               hf8c457e_1001    conda-forge
tk                        8.6.9             h84994c4_1000    conda-forge
tktable                   2.10                 h14c3975_0
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.9             h14c3975_1004    conda-forge
xorg-libsm                1.2.3             h4937e3b_1000    conda-forge
xorg-libx11               1.6.7             h14c3975_1000    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.2             h14c3975_1007    conda-forge
xorg-libxext              1.3.3             h14c3975_1004    conda-forge
xorg-libxrender           0.9.10            h14c3975_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
zlib                      1.2.11            h14c3975_1004    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : R-test
    active env location : /local/conda/envs/R-test
            shell level : 1
       user config file : /some/remote/.condarc
 populated config files : /local/conda/.condarc
                          /some/remote/.condarc
          conda version : 4.5.12
    conda-build version : not installed
         python version : 3.6.6.final.0
       base environment : /local/conda  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/linux-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /some/remote/pkgs_cache
       envs directories : /local/conda/envs
                          /some/remote/conda_envs
               platform : linux-64
             user-agent : conda/4.5.12 requests/2.19.1 CPython/3.6.6 Linux/3.10.0-862.11.6.el7.x86_64 centos/7 glibc/2.17
                UID:GID : 21075:21000
             netrc file : None
           offline mode : False
```
</details>

**Comment by PertuyF** [comment]
This issue was introduced with #61 (see [there](https://github.com/conda-forge/r-base-feedstock/pull/61/files#diff-44a73bcc045c193c3bd45da87994b03bR487)).

**Comment by PertuyF** [comment]
Solved with PR #69
